app/Http/Controllers/DiagnosticsController.php

```diff
@@ -182,27 +206,36 @@ public function get_errors()
 
 		// Check dropboxKey
 		if (!isset($settings['dropbox_key'])) {
-			$errors[] = 'Warning: Dropbox import not working. No property for dropbox_key.';
+			$errors[]
+				= 'Warning: Dropbox import not working. No property for dropbox_key.';
 		} elseif ($settings['dropbox_key'] == '') {
-			$errors[] = 'Warning: Dropbox import not working. dropbox_key is empty.';
+			$errors[]
+				= 'Warning: Dropbox import not working. dropbox_key is empty.';
 		}
 
 		// Check php.ini Settings
-		if (ini_get('max_execution_time') < 200 && ini_set('upload_max_filesize', '20M') === false) {
-			$errors[] = 'Warning: You may experience problems when uploading a large amount of photos. Take a look in the FAQ for details.';
+		if (ini_get('max_execution_time') < 200
```

Comment:
While not directly related to this PR, should we eliminate this check, now that we reset the timeout for each iteration of import or download?

Reply:
sure. but I guess in another PR. :)